k8s/ecr_login.sh

```diff
@@ -0,0 +1,14 @@
+ACCOUNT=553667903818
+REGION=eu-north-1
+SECRET_NAME=${REGION}-ecr-registry
+EMAIL=smthngslv@gmail.com
```

Comment:
Is this a sensative information? 
![HOom](https://user-images.githubusercontent.com/47006007/199178517-7fec310a-46f0-4ca7-9f90-c78bfdb825e5.gif)